build.d/750-geoip-install

```diff
@@ -0,0 +1,36 @@
+#!/usr/bin/env bash
+if [[ "$GEOIP_ACCOUNT_ID" != "" && "$GEOIP_LICENSE_KEY" != "" ]] ; then
+    set -e
+    # see https://dev.maxmind.com/geoip/geoipupdate/
+    echo "Activating GeoIP/GeoLite2 updates"
+    echo "This product includes GeoLite2 data created by MaxMind, available from https://www.maxmind.com"
+    echo "
+# GeoIP.conf file for \`geoipupdate\` program, for versions >= 3.1.1.
+# Used to update GeoIP databases from https://www.maxmind.com.
+# For more information about this config file, visit the docs at
+# https://dev.maxmind.com/geoip/geoipupdate/.
+
+# \`AccountID\` is from your MaxMind account.
+AccountID ${GEOIP_ACCOUNT_ID}
+
+# Replace YOUR_LICENSE_KEY_HERE with an active license key associated
+# with your MaxMind account.
+LicenseKey ${GEOIP_LICENSE_KEY}
+
+# \`EditionIDs\` is from your MaxMind account.
+EditionIDs GeoLite2-City
+    " > /etc/GeoIP.conf
+    curl --silent -L --output geoipupdate_${GEOIP_UPDATER_VERSION}_linux_amd64.deb https://github.com/maxmind/geoipupdate/releases/download/v${GEOIP_UPDATER_VERSION}/geoipupdate_${GEOIP_UPDATER_VERSION}_linux_amd64.deb
+    dpkg -i --force-confold geoipupdate_${GEOIP_UPDATER_VERSION}_linux_amd64.deb
```

Comment:
I see what you're doing here and I wonder... wouldn't it be better to always supply this dependency installed instead, in the base image?

Then just let `geoipupdate` fail or not, depending on if it's properly configured or not.

Reply:
i guess that's a possibility. Would make the base image larger but the updater is under Apache / MIT that is possible. I'll add it to the Dockerfiles then.

Reply:
Yes, please. Whenever possible, it's a decision in this project to inflate the base image, as it makes subimages build faster.

Reply:
I added the geoip updater to the base image.